packages/xo-web/src/xo-app/sr/tab-disks.js

```diff
@@ -41,16 +41,39 @@ import { error } from 'notification'
 const COLUMNS = [
   {
     name: _('vdiNameLabel'),
-    itemRenderer: vdi => (
-      <span>
-        <Text value={vdi.name_label} onChange={value => editVdi(vdi, { name_label: value })} />{' '}
-        {vdi.type === 'VDI-snapshot' && (
-          <span className='tag tag-info'>
-            <Icon icon='vm-snapshot' />
-          </span>
-        )}
-      </span>
-    ),
+    itemRenderer: (vdi, { baseCopiesByVdi }) => {
+      const baseCopies = baseCopiesByVdi[vdi.id]
```

Comment:
See [this](https://github.com/vatesfr/xen-orchestra/pull/5826#discussion_r671309129).